concent_api/concent_api/middleware.py

```diff
@@ -14,3 +18,26 @@ def __call__(self, request):
         response = self.get_response(request)
         response['Concent-Golem-Messages-Version'] = __version__
         return response
+
+
+class ConcentVersionMiddleware(object):
+    """
+    Used to attach version of the Concent currently used to HTTP response header.
+
+    """
+
+    CONCENT_VERSION = None
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        response['Concent-Version'] = self.get_concent_version()
+        return response
+
+    def get_concent_version(self):
```

Comment:
This should be a `@classmethod`. Otherwise you won't be able to get the version without creating an instance.

Reply:
Not exactly, I have added it this way because it was only way I could mock it in tests. This is for middleware usage only.